Hospital/src/patterns/RoomFactory.java

```diff
@@ -0,0 +1,17 @@
+package patterns;
+
+public class RoomFactory {
+	
+	public Room getRoom(String tipo){
+		
+		if (tipo.equals("cuarto comun"))
```

Comment:
[Low] It is a good practice to use {} although when the 'if' block only have one statement. Why? Because we want to avoid the case that you need to add a new statement later for that block, and you forget to add the {}. For example:
```java
if (condition)
  doSomeAction();
```
And then you realized that you need some more action for that condition. You can do the following by mistake:
```java
if (condition)
  doSomeAction();
  otherAction();
```
And in this case, otherAction is always executed, not just when condition is true.